src/client/kotlin/tech/thatgravyboat/skyblockapi/utils/extentions/ItemStackExtensions.kt

```diff
@@ -21,3 +22,9 @@ fun ItemStack.isSameItem(other: ItemStack?): Boolean {
     if (other == null) return false
     return this == other || ItemStack.isSameItemSameComponents(this, other)
 }
+
+fun ItemStack.getTexture(): String? {
+    if (!this.`is`(Items.PLAYER_HEAD)) return null
```

Comment:
technically anything can have a player head texture on it in 1.21.5 so should probably just remove this